### PR TITLE
New Featured Artist: Panda Eyes

### PR DESCRIPTION
**Pull request by Ephemeralis**
🐼

### CODE REVIEW

File: news/2019-11-13-new-featured-artist-panda-eyes.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "New Featured Artist: Panda Eyes"
+date: 2019-11-13 14:00:00 +0000
+---
+
+All eyes are wide open for our latest addition to the osu! Featured Artist lineup - **Panda Eyes**! Check out a selection of **63** new tracks from the artist behind *Highscore*, *Colorblind* and more!
```
Comment (cl8n) [inline news/2019-11-13-new-featured-artist-panda-eyes.md]:
```suggestion
All eyes are wide open for our latest addition to the osu! Featured Artist lineup — **Panda Eyes**! Check out a selection of **63** new tracks from the artist behind *Highscore*, *Colorblind*, and more!
```

File: news/2019-11-13-new-featured-artist-panda-eyes.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+![](https://assets.ppy.sh/artists/55/header.jpg)
+
+Yes, you heard that right — **Panda Eyes** is here, and with a goodly chunk of his discography to boot! From tracks like *Fuck Off* from *KIKO* to hits reminiscent of yesteryear like *Aurora* from *Nostalgia 64*, we've got a huge selection of a sound you most assuredly know and love.
```
Comment (Coppertine) [inline news/2019-11-13-new-featured-artist-panda-eyes.md]: The wording in that paragraph feels like it's used in a game called "Kiko" or "Nostalgia 64" and not albums.

Comment (cl8n) [reply news/2019-11-13-new-featured-artist-panda-eyes.md]: from => in ?

not sure if that helps, I think this is fine as-is though

File: news/2019-11-13-new-featured-artist-panda-eyes.md (hunk 1 above)
Comment (Ephemeralis) [reply news/2019-11-13-new-featured-artist-panda-eyes.md]: oh bother i missed one

File: news/2019-11-13-new-featured-artist-panda-eyes.md (hunk 2 above)
Comment (Ephemeralis) [reply news/2019-11-13-new-featured-artist-panda-eyes.md]: it should be okay as-is, i hope

**Pull request merged**